Add packages and files to learn Spring concepts (4)

LLCCLL opened the pull request:
1.com.test.ex02 package:
To see different results depending on bean scope

2.com.test.ex03 package:
1) To see how to set custom init-method, destroy-method
2) To see how to know bean object's name itself

Google formatter applied
Javadoc not applied

Commit:
Add packages and files to learn Spring concepts (4)

1.com.test.ex02 package:
To see different results depending on bean scope

2.com.test.ex03 package:
1) To see how to set custom init-method, destroy-method
2) To see how to know bean object's name itself

Google formatter applied
Javadoc not applied

Commit:
Add projects to learn Spring concepts (5)

1.springII_Ex01:
  To learn Spring MVC structure.

2.springII_Ex02:
  To see the way to send data with HttpServletRequest, Request param and command object

    Google formatter applied
    Javadoc not applied

Commit:
Add a new project to learn Spring concepts (6)

1) springII_Ex03:
To learn how to transfer datas in form via get and post

Google formatter not applied
Javadoc not applied

Commit:
Add a new project to learn Spring concepts (7)

1) springBBS:
To learn how to get and show data from DB

Google formatter applied
Javadoc not applied

Commit:
Add projects to practice connecting MySQL to Spring

btijnb commented on springBBS_needsMySQL/src/main/webapp/WEB-INF/spring/appServlet/servlet-context.xml:
```diff
@@ -0,0 +1,38 @@
+		<beans:property name="driverClassName" value="oracle.jdbc.driver.OracleDriver" />
+		<beans:property name="url" value="jdbc:oracle:thin:@localhost:1521:orcl"/>
+		<beans:property name="username" value="scott" />
+		<beans:property name="password" value="1234" />
```
@LLCCLL 
---
Mysqlユーザーとパスワードに変更する

btijnb commented on springBBS_needsMySQL/src/main/webapp/WEB-INF/spring/appServlet/servlet-context.xml:
```diff
@@ -0,0 +1,38 @@
+	
+	<beans:bean name="dataSource" class="org.springframework.jdbc.datasource.DriverManagerDataSource">
+		<beans:property name="driverClassName" value="oracle.jdbc.driver.OracleDriver" />
+		<beans:property name="url" value="jdbc:oracle:thin:@localhost:1521:orcl"/>
```
@LLCCLL 
---
環境に合わせて修正

```
jdbc:mysql://localhost:{port}/{dbName}
```

btijnb commented on springBBS_needsMySQL/src/main/webapp/WEB-INF/spring/appServlet/servlet-context.xml:
```diff
@@ -0,0 +1,38 @@
+	</beans:bean>
+	
+	<beans:bean name="dataSource" class="org.springframework.jdbc.datasource.DriverManagerDataSource">
+		<beans:property name="driverClassName" value="oracle.jdbc.driver.OracleDriver" />
```
@LLCCLL 
---
driverClassNameのvalueを修正
```
value="com.mysql.jdbc.Driver"
```

btijnb commented on springBBS_needsMySQL/pom.xml:
```diff
@@ -0,0 +1,165 @@
+			<artifactId>spring-jdbc</artifactId>
+			<version>4.1.4.RELEASE</version>
+		</dependency>
+		       
```
@LLCCLL 
---
依存ファイル追加

```
<dependency>
    <groupId>mysql</groupId>
    <artifactId>mysql-connector-java</artifactId>
    <version>{バージョン}</version>
</dependency>
```

ghost commented:
おかげさまで連動成功しました。
失敗の第一原因は、mysql-connector-javaをダウンロードしなかったことでした。
修正した内容はまたpull requestします。

Commit:
Connect MySQL to Spring to show list

The source codes below must be added into Servers/context.xml in STS.

    <Resource
       auth="Container"
       driverClassName="com.mysql.jdbc.Driver"
       maxActive="50"
       maxWait="1000"
       name="jdbc/scottDB"
       password="spring"
       type="javax.sql.DataSource"
       url="jdbc:mysql://localhost:3306/scottDB"
       username="root"/>

Commit:
Delete temporary project
